Files/Views/SettingsPages/Experimental.xaml

```diff
@@ -70,6 +72,49 @@
                     </local:SettingsBlockControl>
                 </local:SettingsBlockControl.ExpandableContent>
             </local:SettingsBlockControl>
+
+            <TextBlock
+                x:Uid="SettingsDefaultExplorer"
+                Padding="0,12,0,0"
+                FontSize="14"
+                FontWeight="Medium"
+                Text="Default file explorer" />
+
+            <local:SettingsBlockControl
+                x:Uid="SettingsSetAsDefaultExplorer"
+                Title="Set Files as default file manager"
+                HorizontalAlignment="Stretch">
+                <local:SettingsBlockControl.Icon>
+                    <FontIcon Glyph="&#xEC50;" />
+                </local:SettingsBlockControl.Icon>
+                <ToggleSwitch IsOn="{Binding IsSetAsDefaultFileManager, Mode=TwoWay}" Style="{StaticResource RightAlignedToggleSwitchStyle}">
+                    <i:Interaction.Behaviors>
+                        <icore:EventTriggerBehavior EventName="Toggled">
+                            <icore:InvokeCommandAction Command="{Binding SetAsDefaultExplorerCommand, Mode=OneWay}" />
+                        </icore:EventTriggerBehavior>
+                    </i:Interaction.Behaviors>
+                </ToggleSwitch>
+            </local:SettingsBlockControl>
+
+            <local:SettingsBlockControl
+                x:Uid="SettingsSetAsOpenDialog"
```

Comment:
I've been working on moving over to a new method of supporting translations via `x:Bind`, do you think you can use this method as well? https://devblogs.microsoft.com/ifdef-windows/use-a-custom-resource-markup-extension-to-succeed-at-ui-string-globalization/